gradle/libs.versions.toml

```diff
@@ -8,8 +8,8 @@ coroutines = "1.3.8"
 androidXTest = "1.4.0"
 detekt = "1.16.0"
 buildTools = "30.0.3"
-compileSdk = "29"
-targetSdk = "28"
```

Comment:
Preparations for targetSdk = 30.
Make it consistent with avito as a first step.